doorstop/run_bepinex.sh

```diff
@@ -32,7 +32,7 @@ export DOORSTOP_CORLIB_OVERRIDE_PATH=""
 # Special case: program is launched via Steam
 # In that case rerun the script via their bootstrapper to ensure Steam overlay works
 if [ "$2" = "SteamLaunch" ]; then
-    "$1" "$2" "$3" "$4" "$0" "$5"
+    "$1" "$2" "$3" "$4" "$5" "$6" "$0" "$7"
```

Comment:
@ManlyMarco I don’t have a Linux Steam to test this on, but if you just want to forward all arguments—that is, assuming that there aren’t additional arguments that Steam passes in but you want to drop—then this should be compatible with old and new versions alike:

```
count=0
until test $((count+=1)) -ge $#
do
    set -- "$@" "$1"
    shift
done
set -- "$@" "$0" "$1"
shift
"$@"
```

(Thanks to https://stackoverflow.com/a/62227833/865331 for a cross-platform implementation.)

Also, rather than `"$@"; exit` in this line and the line after it, you may want to `exec "$@"`.

Want me to submit a pull request?

Reply:
> Want me to submit a pull request?

Yes, please do, that would be great. It'll have to be applied to the master branch as well.

Reply:
Hmm, the version on master passes the remaining arguments through, as though they can be non-empty.  This originates in #391 without any apparent explanation. If extra arguments are really possible, then this should rather parse the command line, find the `steam-launch-wrapper [--]` and insert BepInEx after that, and it probably needs to preserve the extra arguments in v5, too. Alternatively, I guess it can just count the same fixed 5 arguments but avoid counting any instances of `--`.